Apps/Laboratory/src/Delegates/RestLaboratoryDelegate.cs

```diff
@@ -158,20 +167,21 @@ public async Task<RequestResult<LaboratoryReport>> GetLabReport(Guid id, string
                 client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("bearer", bearerToken);
                 client.DefaultRequestHeaders.Accept.Add(
                 new MediaTypeWithQualityHeaderValue(MediaTypeNames.Application.Json));
-                var query = new Dictionary<string, string?>
+                Dictionary<string, string?> query = new()
                 {
                     ["subjectHdid"] = hdid,
                 };
                 try
                 {
-                    Uri endpoint = new Uri(QueryHelpers.AddQueryString($"{this.labConfig.Endpoint}/{id}/LabReportDocument", query));
+                    string endpointString = $"{this.labConfig.BaseUrl}{this.labConfig.LabOrdersEndpoint}/{id}/LabReportDocument";
+                    Uri endpoint = new(QueryHelpers.AddQueryString(endpointString, query));
                     HttpResponseMessage response = await client.GetAsync(endpoint).ConfigureAwait(true);
                     string payload = await response.Content.ReadAsStringAsync().ConfigureAwait(true);
                     this.logger.LogTrace($"Get laboratory report response payload: {payload}");
                     switch (response.StatusCode)
                     {
                         case HttpStatusCode.OK:
-                            var options = new JsonSerializerOptions
+                            JsonSerializerOptions options = new()
```

Comment:
Do we need these?  We had been killing off the serializer options.

Reply:
I'm not sure.  This was existing code.